tests/global.setup.ts

```diff
@@ -24,7 +24,7 @@ async function globalSetup() {
   await vscodeApp.closeVSCode();
   console.log('Completed global setup.');
 
-  if (getOSInfo() === 'windows' && process.env.CI) {
+  if (getOSInfo() === 'windows' || process.env.CI) {
```

Comment:
This will also add extra step to the linux pipeline, which can run faster and don't need this.

When we're running the tests locally on Windows we can always update these values so therei s no need to commit them.

The same applies to the previous change

Reply:
done